altdoc/about.md

```diff
@@ -0,0 +1,38 @@
+## Developers
+
+The primary developer of the upstream Polars project is Ritchie Vink
+([@ritchie46](https://github.com/ritchie46)). This R package is
+maintained by [@eitsupi](https://github.com/eitsupi) and [Etienne Bacher](https://github.com/etiennebacher). The original R package was created by Søren Welling
+([@sorhawell](https://github.com/sorhawell)).
```

Comment:
I do not know if these information is necessary.
(In my opinion I would like to remove)

Reply:
Why do you want to remove that? `pkgdown` lists authors in the footer but we don't have that, so this page seems relevant to me. I also think it makes sense to mention the upstream project and to give credit to the original implementation, which is not really visible in the `DESCRIPTION`.

Reply:
- The binary contains a number of upstream Rust packages, not just polars. For example, CRAN requires that the authors of all dependencies be included[^1].
- The original author of this package is not actually Søren. I did not start writing this package based on what Søren wrote.

[^1]: <https://cran.r-project.org/web/packages/using_rust.html>

Reply:
Just to clarify, I'm only talking about mentioning those in the docs, I'm not talking about complying with certain CRAN standards. It simply seems fair to me to mention the original implementation upstream since it is obviously the reference of this package.

> The original author of this package is not actually Søren. I did not start writing this package based on what Søren wrote. 

I'm not saying it is, but similarly as above mentioning it as a sort of acknowledgement seems natural to me. It's a bit similar to listing "Existing work" or "Alternatives" in a README for instance.

Reply:
Since this file is not part of the package, I think such a thing should be in the README or something similar and not in a website-specific file.

Reply:
I really don't understand the problem with this page. It's just an "About" page where we list current and previous core contributors to the package. What bothers you about it?

> Since this file is not part of the package, I think such a thing should be in the README or something similar and not in a website-specific file.

I don't understand this, AFAIK the README is also not included in the package (looking at package folders listed in `.libPaths()`) so how is it different from a website-specific file?

Reply:
The README is included in the source package and is displayed on CRAN, R-universe, etc.

Package information can be found here.
https://github.com/eitsupi/neo-r-polars/blob/5d3b2cafc7693c49fbd341fdb1283514dce112ac/man/neopolars-package.Rd#L11-L18